wagtail_shortcode/wagtail_hooks.py

```diff
@@ -1,26 +1,91 @@
-from django.urls import include, path
-from django.views.i18n import JavaScriptCatalog
+from django.conf import settings
+from django.utils.html import json_script
+from django.utils.safestring import mark_safe
+
+# TODO confirm version requirements
```

Comment:
**TODO/Issue to resolve**

I'll remove this comment and go with what ever cookiecutter-wagtail-package supports.